GOVERNANCE.md

```diff
@@ -18,129 +26,133 @@ the cert-manager GitHub org must follow the guidelines in [the contributing
+GitHub organization.
+
+### Becoming a GitHub Member
+
+To be added as a GitHub member of the cert-manager organization, you will need
+to look for two sponsors with at least the `reviewer` role. These two sponsors
+must have had some meaningful interaction with you on an issue on GitHub or on
+the cert-manager or cert-manager-dev channels on Slack.
+
+Then, open an issue on the `cert-manager/cert-manager` repository and mention
+the sponsors as well as links to the meaningful interations (Slack threads,
+GitHub issues). Ask your sponsors to confirm their sponsorship by commenting on
+your PR. After that, your request will be reviewed by a cert-manager admin, in
+accordance with their SLO.
+
+To be added as a GitHub member, you will also need to enable [two-factor authentication][] on your GitHub account.
+
+GitHub members are encouraged to engage with the [cert-manager-dev][] mailing list as well as the [cert-manager](https://kubernetes.slack.com/messages/cert-manager) and [cert-manager-dev](https://kubernetes.slack.com/messages/cert-manager-dev) Slack channels.
 
 [two-factor authentication]: https://help.github.com/articles/about-two-factor-authentication
 [cert-manager-dev]: https://groups.google.com/forum/#!forum/cert-manager-dev
 
-### Privileges
-- Is a member of the cert-manager GitHub organization
+## Reviewers
 
-### Responsibilities
-- Remain an active contributor to the project. If you have not been able to contribute to the project in longer than 18 months, you will be removed from the organization. You must respond to any raised member inactivity issues on the cert-manager/org repo in which you are @mentioned as the inactive member within 30 days to prove your activity.
-- Monitor the cert-manager-dev and cert-manager-dev channels on Slack, and help out when possible.
+The mission of the reviewer is to read through PRs for quality and correctness
+on all or some part of cert-manager. Reviewers are knowledgeable about the
+codebase as well as software engineering principles. Reviewers are defined in
+the file [`OWNERS`](./OWNERS).
+
+### Becoming a Reviewer
 
-## Reviewer Role
+To become a reviewer, you will need to look for a sponsor with at least the
+`approver` role. Then, create a PR to add your name to the list of `reviewers`
+in the `OWNERS` file. The PR description should list your significant
+contributions.
 
-Reviewers are able to review code for quality and correctness on some part of cert-manager.  
-They are knowledgeable about both the codebase and software engineering principles.
+Your sponsor must have the approver role. Your sponsor must have had close
+interactions with you: he must have been closely reviewed one of your PRs or
+worked with you on a thorny issue. The sponsor is expected to give his approval
+as a comment on the `OWNERS` PR. Additionally, your `OWNERS` PR should list your
+substantial contributions to the project.
 
-**Defined by:** `reviewers` entry in the cert-manager [OWNERS](./OWNERS) file
```

Comment:
@maelvls could we keep these **Defined by:** sections? I think that for these  sections it is quite useful to have them seperate from the text description.